Documentation/network/bgp-control-plane.rst

```diff
@@ -446,6 +530,34 @@ The following command shows peering status:
      -D, --debug           Enable debug messages
      -H, --host string     URI to server-side API
 
+The following command shows BGP routes available in the RIB / advertised to the peers:
+
+.. code-block:: shell-session
+
+   cilium# cilium bgp routes -h
```

Comment:
Consider adding a note before the code block that states this is being run from `cilium-dbg` in the agent container.

Reply:
Actually, this should be stated as `cilium-dbg` in the docs, fixing.